policy_definitions/tagging/policy_assignment_qby_deploy_require_tags.tmpl.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "QBY-Require-Tags",
+    "type": "Microsoft.Authorization/policyAssignments",
+    "apiVersion": "2022-06-01",
+    "dependsOn": [],
+    "properties": {
+      "description": "Automatically generated policy set to enforce existence of a tag on resource groups and all resources.",
```

```suggestion
      "description": "Enforce existence of conceptual defined tags.",
```
This is the description the user will see in the portal. It should include more information then just the set.

---

Fix as suggestion